quantity.hpp

```diff
@@ -63,6 +71,218 @@ using    day_scaling = std::ratio_multiply<  hour_scaling, scale<24>>;
+
+
+template<typename Tag, int Power>
+struct  pow_tag : create_tag<typename Tag::id, std::ratio_multiply<typename Tag::exponent,std::ratio<Power>>::num
+                                             , std::ratio_multiply<typename Tag::exponent,std::ratio<Power>>::den > {};
+
+
+template<typename Tag1, typename Tag2>
+struct are_tags_compatible;
+template< typename ID1, std::intmax_t N1, std::intmax_t D1
+        , typename ID2, std::intmax_t N2, std::intmax_t D2 >
+struct are_tags_compatible<tag<ID1,N1,D1>,tag<ID2,N2,D2>> : std::integral_constant<bool, tag<ID1,N1,D1>::is_no_tag::value
+	                                                                                  or tag<ID2,N2,D2>::is_no_tag::value
+	                                                                                  or detail::is_same_v<ID1,ID2> > {
+};
+template<typename Tag1, typename Tag2>
+struct are_tags_compatible : are_tags_compatible< typename create_tag<Tag1>::type
+	                                            , typename create_tag<Tag2>::type > {};
+
+}
+
+/**
+ * @brief Tag creation
+ *
+ * This creates a tag from an ID and a ratio.
+ *
+ * @tparam  ID  tag ID
+ * @tparam Num  tag ratio numerator
+ * @tparam Den  tag ratio denominator
+ *
+ * @note If you pass a tag as the ID parameter, Num and Dem will be ignored
+ *       and the tag returned instead. This allows this meta function to be
+ *       called with either a tag ID or a tag and always return a valid tag,
+ *       in turn allowing users to provide tag names only, rather than to
+ *       create tags each time.
+ */
+template<typename ID, std::intmax_t Num = 1, std::intmax_t Den = 1>
+using create_tag_t = typename detail::create_tag<ID,Num,Den>::type;
+
+/**
+ * @{
+ *
+ * @brief Tag compatibility
+ *
+ * Check whether two tags are compatible. If either or both parameters are tag
+ * names, rather than tags, tags will be created from them before checking
+ * them.
+ *
+ * @tparam Tag1  tag to check for compatibility
+ * @tparam Tag2  tag to check for compatibility
+ *
+ * @note Two tags are compatible if they have the same ID or if either is
+ *       the no_name tag.
+ */
+template<typename Tag1, typename Tag2>
+using are_tags_compatible = detail::are_tags_compatible<Tag1, Tag2>;
+template<typename Tag1, typename Tag2>
+constexpr bool are_tags_compatible_v = are_tags_compatible<Tag1, Tag2>::value;
+/** @} */
+
+
+/**
+ * @{
+ *
+ * @brief Get tag resulting from multiplying/dividing quantities
+ *
+ * This will calculate the tags resulting from multiplying/dividing quantities.
+ *
+ * The resulting tag is defined thus:
+ *
+ *  1. If both tags are the `no_tag`, then that is the resulting tag, too.
+ *  2. If either of the tags is `no_tag`, the resulting tag will be the other
+ *     tag.
+ *  3. If both tags are equal, the resulting tag ID will be the same, their
+ *     ratios added/subtracted.
```

Comment:
```suggestion
 *     exponent ratios added/subtracted.
```

Reply:
I searched for _"ratio"_ in the documentation and replaced all occurrences I found with _"exponent"_.